generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,533 @@
+
+## Other Link Functions
+
+Logistic regression (regression using a logit link function) works well for binary outcomes.
+But what about other kinds of outcome variables?
+
+If your outcome is counts of something, you can use a **Poisson** or **Negative Binomial** link function. 
+For example, perhaps your outcome is the number of [Adverse Childhood Experiences (ACEs)](https://www.cdc.gov/violenceprevention/aces/fastfact.html) per patient. 
+It's not possible for someone to have fewer than 0 ACEs, so that variable is bound at 0. 
+It's also measured as a count (how many total ACEs experienced), so no one will have 1.2 ACEs, for example. 
+
+For outcomes bounded at 0 and 1 (like a probability or proportion), you can use logistic regression, but a **beta** link function may be more appropriate.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For more details on beta regression and when to use it, see [Analysing continuous proportions in ecology and evolution: A practical introduction to beta and Dirichlet regression](https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.13234) by Douma and Weedon (2019).
+
+</div>
+
+If your outcome is positively skewed, especially something bound at 0, then you can use a **Gamma** or **Inverse-Gaussian** link function. 
+These are especially useful for outcomes like cost, income, or length of stay -- continuous variables bound at 0 and usually positively skewed.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+To learn more about how to talk about the shape of a variable, such as identifying positive and negative skew, see the [Khan Academy lesson on shapes of distributions](https://www.khanacademy.org/kmap/measurement-and-data-j/md231-data-distributions/md231-displays-of-distributions/v/shapes-of-distributions).
+
+</div>
+
+## Special Considerations for Generalized Linear Models
+
+When you run generalized linear models, it feels very similar in many ways to running a regular linear model. 
+Your output will look largely similar (coefficient tests, measurement of overall model fit, etc.). 
+Under the hood, though, your statistical software is doing something very different.
+
+Unlike linear models, there is not a single analytic solution to generalized linear models -- that means you couldn't (even if you wanted to) figure out the solution by solving the equation by hand. 
+Instead, the computer uses brute force to find a solution, iterating over tons of combinations of parameter estimates and seeing which gives the best model fit.
+
+This means the results are not exact --- they depend on the sampling algorithm being used --- so you might see very small changes if you run the model again, or if you run it on other software. 
+There's also a risk that your model won't converge if it's too complicated and you don't have enough data.
```

Comment:
Do we need to insert a brief sidebar about what it means for a model to converge (or, alternatively just use simpler phrasing here)? Its such a brief note that I could be convinced this is fine, but wanted to flag that it could be a bit too jargony for someone who doesn't run many models.

Reply:
```suggestion
There's also a risk that your model won't converge (meaning you won't get any results at all) if it's too complicated and you don't have enough data. 
```

Not anything close to a definition or explanation of convergence, but I'm trying not to get myself into the weeds here :) Does this seem like a good balance between clarity and brevity?

Reply:
Love it!